.konflux/olm-catalog/bundle/Dockerfile

```diff
@@ -1,10 +1,18 @@
 # use non-scratch parent image as dummy for https://issues.redhat.com/browse/KFLUXBUGS-10
-FROM registry.redhat.io/ubi8/ubi-micro:latest
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest as base
+
+COPY . .
+RUN curl -Lo /usr/local/bin/yq https://github.com/mikefarah/yq/releases/download/v4.45.1/yq_linux_amd64 && \
```

Comment:
The file shouldn't be downloaded I think, it will be downloaded in the context and should be available when you do `COPY . .`.

Reply:
See https://github.com/containerbuildsystem/cachi2/blob/main/docs/generic.md#using-fetched-dependencies and https://konflux.pages.redhat.com/docs/users/how-tos/configuring/prefetching-dependencies.html#generic